README.md

```diff
@@ -107,6 +107,7 @@ All modules are disabled by default and need to be activated explicitly in your
 lua <<EOF
 require'nvim-treesitter.configs'.setup {
   ensure_installed = "maintained", -- one of "all", "maintained" (parsers with maintainers), or a list of languages
+  sync_install = true, -- install languages synchronously
```

Comment:
I think the example on the readme should be set to `false`, since most people will just copy/paste it.

Reply:
Fair point, updated.